components/common/ReactDayPicker/ReactDayPicker.jsx

```diff
@@ -51,6 +51,7 @@ const useStyles = makeStyles(theme => ({
 
     '& .DayPicker-Day--disabled': {
       color: colors.textSecondaryDark,
+      pointerEvents: 'none !important',
```

Comment:
```suggestion
      pointerEvents: 'none',
```

Pretty sweet that resolving this issue boils down to just a single CSS rule! `!important` isn't necessary here though; nothing else sets `pointer-events` on this element